### PR TITLE
[stable/rabbitmq] Allow specifying ulimit and update healthchecks to avoid high CPU usage

### PR DESCRIPTION
**Pull request by tompizmor**
Signed-off-by: tompizmor <tompizmor@gmail.com>

#### What this PR does / why we need it:
This PR allows to specify a value for the max file descriptors for rabbitmq via the `rabbitmq.ulimitNofiles` and it also updates the healtchecks to not use the command `rabbitmqctl status` to avoid high CPU usage: https://github.com/helm/charts/issues/3855

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - https://github.com/helm/charts/issues/3855

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq/values.yaml [hunk 1]
```diff
@@ -29,12 +29,12 @@ rbacEnabled: true
 ## section of specific values for rabbitmq
 rabbitmq:
    ## RabbitMQ application username
-  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq/blob/master/README.md#creating-a-database-user-on-first-run
+  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq/blob/master/README.md#environment-variables
```
Comment (juan131) [inline stable/rabbitmq/values.yaml]: Fix the links on `values-production.yaml` too

**Review by juan131 — APPROVED** [review]
Great thanks!

**Comment by intelfx** [comment]
They say it's wrong to use `/api/healthchecks/node` as you do: https://github.com/helm/charts/issues/7752

File: stable/rabbitmq/templates/statefulset.yaml [hunk 2]
```diff
@@ -89,9 +88,7 @@ spec:
         {{- if .Values.livenessProbe.enabled }}
         livenessProbe:
           exec:
-            command:
-            - rabbitmqctl
-            - status
+            command: ["sh", "-c", "curl -f --user {{ .Values.rabbitmq.username }}:$RABBITMQ_PASSWORD 127.0.0.1:{{ .Values.rabbitmq.managerPort }}/api/healthchecks/node"]
```
Comment (thomas-riccardi) [inline stable/rabbitmq/templates/statefulset.yaml:91]: this endpoints returns 200 even if it detects an error, see https://github.com/helm/charts/issues/7752

**Comment by tompizmor** [comment]
@intelfx @thomas-riccardi please check this other PR https://github.com/helm/charts/pull/8146

**Comment by alexsandro-xpt** [comment]
@tompizmor this PR should be accept and released to after can I run `helm install stable/rabbitmq` again?

**Comment by tompizmor** [comment]
Hi @alexsandro-xpt!

The PR is already merged so you can run `helm repo update` and then `helm install`. Just make sure that you are installing the 3.3.1 version or newer.

**Comment by alexsandro-xpt** [comment]
@tompizmor  I run `helm install --name my-rabbitmq --set livenessProbe.enabled=false,readinessProbe.enabled=false,rabbitmq.username=admin,rabbitmq.password=mypwd,rabbitmq.erlangCookie=secretcookie  stable/rabbitmq` to fix it.

**Pull request merged**